packages/core/src/codewhisperer/util/runtimeLanguageContext.ts

```diff
@@ -106,7 +108,10 @@ export class RuntimeLanguageContext {
         })
         this.supportedLanguageExtensionMap = createConstantMap<string, CodewhispererLanguage>({
             c: 'c',
+            h: 'c',
             cpp: 'cpp',
+            cc: 'cpp',
+            'c++': 'cpp',
```

Comment:
to align with JB impl https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/language/CodeWhispererLanguageManager.kt#L120-L149